aerial_robot_control_name: aerial_robot_control/manipulator_controller

controller:
  end_effector_name: link4_tip
  transform_duration: 1.0
